samples/snippets/cloud-client/src/main/java/privateca/CreateCertificate.java

```diff
@@ -0,0 +1,186 @@
+      String commonName = "common-name";
+      String orgName = "org-name";
+      String domainName = "dnsname.com";
+      long certificateLifetime = 1000L;
+
+      // Set the Public Key and its format as obtained from the Cloud KMS.
+      PublicKey publicKey =
+          PublicKey.newBuilder().setKey(publicKeyBytes).setFormat(KeyFormat.PEM).build();
+
+      SubjectConfig subjectConfig =
+          SubjectConfig.newBuilder()
+              // Set the common name and org name.
+              .setSubject(
+                  Subject.newBuilder().setCommonName(commonName).setOrganization(orgName).build())
+              // Set the fully qualified domain name.
+              .setSubjectAltName(SubjectAltNames.newBuilder().addDnsNames(domainName).build())
+              .build();
+
+      // Set the X.509 fields required for the certificate.
+      X509Parameters x509Parameters =
+          X509Parameters.newBuilder()
+              .setKeyUsage(
+                  KeyUsage.newBuilder()
+                      .setBaseKeyUsage(
+                          KeyUsageOptions.newBuilder()
+                              .setDigitalSignature(true)
+                              .setKeyEncipherment(true)
+                              .setCertSign(true)
+                              .build())
+                      .setExtendedKeyUsage(
+                          ExtendedKeyUsageOptions.newBuilder().setServerAuth(true).build())
+                      .build())
+              .setCaOptions(CaOptions.newBuilder().setIsCa(true).buildPartial())
+              .build();
+
+      // Create certificate.
+      Certificate certificate =
+          Certificate.newBuilder()
+              .setConfig(
+                  CertificateConfig.newBuilder()
+                      .setPublicKey(publicKey)
+                      .setSubjectConfig(subjectConfig)
+                      .setX509Config(x509Parameters)
+                      .build())
+              .setLifetime(Duration.newBuilder().setSeconds(certificateLifetime).build())
+              .build();
+
+      // Create the Certificate Request.
+      CreateCertificateRequest certificateRequest =
+          CreateCertificateRequest.newBuilder()
+              .setParent(CaPoolName.of(project, location, caPoolName).toString())
+              .setCertificateId(certificateName)
+              .setCertificate(certificate)
+              .setIssuingCertificateAuthorityId(certificateAuthorityName)
+              .build();
+
+      // Get the Certificate response.
+      ApiFuture<Certificate> future =
+          certificateAuthorityServiceClient
+              .createCertificateCallable()
+              .futureCall(certificateRequest);
+
+      // Get the PEM encoded, signed X.509 certificate.
+      // response.getPemCertificate()
+      // To verify the obtained certificate, use this intermediate chain list.
+      // response.getPemCertificateChainList()
+      Certificate response = future.get();
```

Comment:
are we doing anything with this response?

Reply:
Fixed it to sysout certificate.